oci/archive/oci_transport.go

```diff
@@ -171,18 +172,24 @@ func createOCIRef(sys *types.SystemContext, image string) (tempDirOCIRef, error)
 
 // creates the temporary directory and copies the tarred content to it
 func createUntarTempDir(sys *types.SystemContext, ref ociArchiveReference) (tempDirOCIRef, error) {
-	tempDirRef, err := createOCIRef(sys, ref.image)
-	if err != nil {
-		return tempDirOCIRef{}, fmt.Errorf("creating oci reference: %w", err)
-	}
 	src := ref.resolvedFile
-	dst := tempDirRef.tempDirectory
 	// TODO: This can take quite some time, and should ideally be cancellable using a context.Context.
```

Comment:
When moving things around, this should probably end up just before `Untar()` below. The `Open` is not expected to take that long.

Reply:
… this _comment_ should …